### ISSUE TITLE
Contentful Speed Index generates errors

### ISSUE THREAD

**Issue by soulgalore** [sitespeedio/browsertime#1013]
Hi @dpalmeiro could you help me with this? I've got some reports on Slack of errors from Visual Metrics that I haven't been able to reproduce until now. I got a video file and the log. The error:

```
Traceback (most recent call last):
  File "./visualmetrics.py", line 1875, in main
    options.herodata)
  File "./visualmetrics.py", line 1324, in calculate_visual_metrics
    'value': calculate_contentful_speed_index(progress, dirs)})
  File "./visualmetrics.py", line 1477, in calculate_contentful_speed_index
    value  = int(output.split()[7].split(':')[0])
IndexError: list index out of range
```
I can reproduce it like this: 
`$browsertime/vendor ./visualmetrics.py -i error.mp4  --orange --perceptual --contentful --force --renderignore 5 --json --viewport -vvv`

Removing the Contentful switch makes it work again.

https://send.firefox.com/download/705cdf51d7b31083/#SFzPnGSrl8GqP_Hi1APP6g

Best
Peter

**Comment by soulgalore** [comment]
I wrapped the code in a try/catch now and then we can take on the real issue in the future. This makes the rest of the Visual Metrics work.

**Comment by soulgalore** [comment]
This is fixed in 8.0.0
